docs/api/completers/argparser.rst

```diff
@@ -1,10 +1,10 @@
-.. _xonsh_completers_xompletions:
+.. _xonsh_completers_argparser:
 
 *********************************************************
-Completion Xompletions (``xonsh.completers.xompletions``)
+Completion for argparser based commands (``xonsh.completers.argparser``)
```

Comment:
RST likes for underlines and overlines to have the same character count as the enclosed line -- can you add more `*`s?

Reply:
I will hold this PR for now. I want to move the completer class to cli_utils.py too.